totalRP3/UI/Browsers/MusicBrowser.lua

```diff
@@ -0,0 +1,160 @@
+-- Copyright The Total RP 3 Authors
+-- SPDX-License-Identifier: Apache-2.0
+
+local L = TRP3_API.loc;
+
+local LRPM12 = LibStub:GetLibrary("LibRPMedia-1.2");
+
+local function GenerateSearchableString(str)
+	return string.utf8lower(string.trim((string.gsub(str, "%p+", " "))));
```

Comment:
As per #1090 there was mention of lower being enough, don't know if you care about replacing it here or if you're just waiting for it to be centralized

Reply:
Yeah just waiting.